mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -622,60 +637,58 @@ export const Columns = (props: ColumnsProps) => {
         };
 
         return (
-          <VerticalContainer>
-            <ol className={styles.comments}>
-              {row.original.comments
-                .map((comment) => (
-                  <li
-                    key={comment.uuid}
-                    className={styles.comment}
-                  >
-                    <HorizontalContainer key={comment.uuid}>
-                      <VerticalContainer>
-                        <HorizontalContainer className={styles.horizontalContainer}>
-                          <Link
-                            value={getName(props.way, comment.ownerUuid)}
-                            path={pages.user.getPath({uuid: comment.ownerUuid})}
-                          />
-                          {comment.ownerUuid === user?.uuid &&
-                            <Tooltip
-                              content="Delete comment"
-                              position={PositionTooltip.LEFT}
-                            >
-                              <TrashIcon
-                                className={styles.icon}
-                                onClick={() => renderModalContent({
-                                  description: `Are you sure that you want to delete comment "${comment.description}"?`,
-
-                                  /**
-                                   * CallBack triggered on press ok
-                                   */
-                                  onOk: () => deleteComment(comment.uuid),
-                                })
-                                }
-                              />
-                            </Tooltip>
+          <VerticalContainer className={styles.list}>
+            {row.original.comments
```

Comment:
very hard to read. simplify

Reply:
Let's create task for refactoring please

Reply:
ok, don't hesitate to create any task you want, just manage it

Reply:
Task #371